website/source/docs/enterprise/autopilot/index.html.md

```diff
@@ -38,5 +38,5 @@ completely lost, only one voter will be lost, so the cluster remains available.
 If a voter is lost in an availability zone, Autopilot will promote the non-voter 
 to voter automatically, putting the hot standby server into service quickly.
 
-See the [Nomad Autopilot Guide](https://www.nomadproject.io/guides/cluster/autopilot.html)
+See the [Nomad Autopilot Guide](https://www.nomadproject.io/guides/autopilot.html)
```

Comment:
Any particular reason this is an absolute link where the others are relative?

Reply:
Yeah this should be relative